### PR TITLE
#136 Removed Unused Variable from app.js

### LINKED ISSUE #136: Remove unused variables from base.js
Most of the variables in the code are defined inside `elements` object in `base.js`. Check that all of them are being used somewhere outside `base.js` and if not, remove them. (for eg: `getBookmarks` is redundant). 
You can complete this task easily by using the search feature of any modern editor.

### PR DESCRIPTION
**Pull request by keshav2212**
Fixes #136

**Description**
Removed Unused Variable from app.js
.
**Checklist:**
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

### CODE REVIEW

**Review by makkoncept — CHANGES_REQUESTED** [review]
@keshave2212 Looks good. Refer to the comment below about a minor spelling error. Also, can you please confirm that you have checked all the variables defined in the elements object?

Also, check the appropriate options in the Checklist of PR.

File: src/popup/base.js [hunk 1]
```diff
@@ -42,7 +41,6 @@ export const elements = {
   deleteAllBookmarksButton: document.querySelector('.delete-all-button'),
   buttonBackToTop: document.getElementsByClassName('button-backToTop')[0],
   exportBookmarkskButton: document.getElementsByClassName('export-bookmarks-button')[0],
```
[the remark this answers is not in the document]
Comment (keshav2212) [reply src/popup/base.js]: Done.

[the remark this answers is not in the document]
Comment (keshav2212) [reply src/popup/base.js]: @makkoncept I am not getting how to confirm you that I have gone through all the elements. Can you suggest something ?

[the remark this answers is not in the document]
Comment (makkoncept) [reply src/popup/base.js]: @keshav2212 No, I just wanted to confirm from you. Looks like you have!

**Review by makkoncept — APPROVED** [review]
Looks good! thanks for working.

**Comment by keshav2212** [comment]
@makkoncept Thanks for helping out in my first contribution towards CC.

**Pull request merged**